### ISSUE TITLE
[FEATURE-REQUEST]: Change fly speed boost by scrolling

### ISSUE THREAD

**Issue by BigBrainRobin29** [NoRiskClient/issues#664]
### Your idea

Title, wie z.B. beim Flashback replay viewer.

### Where would this be applied?

Client

### What platforms would this affect?

Not a launcher feature

**Comment by SirKnubble** [comment]
#664

**Comment by BigBrainRobin29** [comment]
@SirKnubble ?

**Comment by SirKnubble** [comment]
> @SirKnubble ?

#643 ich bin dumm

**Comment by BigBrainRobin29** [comment]
Ja, ich mein aber dass man wenn man Ctrl drückt erstmal auf der im Module eingestellten Geschwindigkeit startet, aber dann mit scrollen die Geschwindigkeit ändern kann, wie schon gesagt so wie beim Flashback Replay Viewer

**Comment by BigBrainRobin29** [comment]
Brauchen wir das Limit von 8x?
Und fände glaub gut wenn man die steps beim scrollen einstellen kann oder so, manche wollen das maybe schneller ändern (oder langsamer kp)
